presto-hive/src/main/java/com/facebook/presto/hive/HivePageSink.java

```diff
@@ -142,6 +143,7 @@ public HivePageSink(
             boolean respectTableFormat,
             int maxOpenPartitions,
             boolean immutablePartitions,
+            boolean compressed,
```

Comment:
Should this be `compress` rather than `compressed` since it is "telling it what to do" rather than stating a fact?